.github/workflows/editor.yaml

```diff
@@ -4,105 +4,52 @@ on:
-          else
-            echo "no_changes=false" >> $GITHUB_OUTPUT
-          fi
-
-      - name: Commit changes to the new branch
-        if: steps.check_published.outputs.already_published == 'false' && steps.check_changes.outputs.no_changes == 'false'
-        run: |
-          NEXT_VERSION=$(node -p "require('./packages/editor/package.json').version")
-          git add .
-          git commit -m "chore(release): bump @serlo/editor to version $NEXT_VERSION"
-          git push origin "release/@serlo/editor-$NEXT_VERSION"
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Create Pull Request
-        if: steps.check_published.outputs.already_published == 'false' && steps.check_changes.outputs.no_changes == 'false'
-        id: create_pr
-        run: |
-          NEXT_VERSION=$(node -p "require('./packages/editor/package.json').version")
-          PR_URL=$(gh pr create --title "Release: Bump @serlo/editor to version $NEXT_VERSION" --body "Automated PR to bump @serlo/editor to $NEXT_VERSION" --base staging --head "release/@serlo/editor-$NEXT_VERSION" --json url -q .url)
-          echo "pr_url=$PR_URL" >> $GITHUB_OUTPUT
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Wait for PR to be merged
-        if: steps.create_pr.outputs.pr_url
-        run: |
-          PR_URL="${{ steps.create_pr.outputs.pr_url }}"
-          while true; do
-            PR_STATE=$(gh pr view $PR_URL --json state -q .state)
-            if [ "$PR_STATE" = "MERGED" ]; then
-              echo "PR has been merged"
-              break
-            elif [ "$PR_STATE" = "CLOSED" ]; then
-              echo "PR was closed without merging"
-              exit 1
-            fi
-            echo "Waiting for PR to be merged..."
-            sleep 60
-          done
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Publish to npm (after PR merge)
-        if: steps.check_published.outputs.already_published == 'false' && steps.create_pr.outputs.pr_url
         run: yarn editor:publish
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
-      - name: Create GitHub Release (after PR merge)
-        if: steps.check_published.outputs.already_published == 'false' && steps.create_pr.outputs.pr_url
+      - name: Create and push Git tag
+        if: steps.check_published.outputs.already_published == 'false'
         run: |
-          NEXT_VERSION=$(node -p "require('./packages/editor/package.json').version")
-          body=$(git log -1 --pretty=format:%B)
-          gh release create "v$NEXT_VERSION" --notes "$body"
+          VERSION=$(node -p "require('./packages/editor/package.json').version")
+          git tag v$VERSION
+          git push origin v$VERSION
+
+      - name: Create GitHub Release
+        if: steps.check_published.outputs.already_published == 'false'
+        uses: actions/create-release@v1
```

Comment:
This action is not maintained anymore. You could use one of the maintained actions that are listed in the readme of [actions/create-release](https://github.com/actions/create-release).

I thought you would like to have a changelog also. Do you plan to do it afterwards?

Reply:
github action runner pick it up automatically